packages/mermaid/package.json

```diff
@@ -1,11 +1,19 @@
 {
-  "name": "@dmt/mermaid",
+  "name": "@development-framework/mermaid",
   "version": "1.0.0",
+  "license": "MIT",
   "main": "src/index.tsx",
-  "dependencies": {
-    "mermaid": "^9.1.3"
+  "devDependencies": {
+    "@types/react": "^17.0.39",
+    "typescript": "^4.5.5",
+    "@development-framework/dm-core": ">=1.0.5",
```

Comment:
Is this needed as a devDependency when it's already defined as a _direct_ dependency?

Reply:
I'd argue cleaning up these package dependencies is a separate issue/PR :)

Reply:
added to "direct" dependencies